app/src/main/java/com/example/travelapp/BinaryMinHeap.java

```diff
@@ -0,0 +1,134 @@
+package com.example.travelapp;
```

Comment:
as a note, if you copy/pasted from somewhere for any part of this, add a comment of where you got the code from